src/aind_behavior_curriculum/base.py

```diff
@@ -0,0 +1,44 @@
+"""
+Base behavior pydantic object
+"""
+
+from pydantic import BaseModel, ConfigDict
+
+
+class AindBehaviorModel(BaseModel):
+    """
+    Defines Pydantic configurations applied to all behavior models.
+    BaseModel: Validate arguments on initialization.
+    Configurations:
+        - extra='forbid':
+            Do not allow a model to be initalized with undocumented parameters.
+        - validate_assignment=True:
+            Revalidate fields against schema on any change to model instance.
+        - validate_defaults=True:
+            Validate default fields on subclasses.
+        - strict=True:
+            Enforce strict typing.
+    """
+
+    model_config = ConfigDict(
+        extra="forbid",
```

Comment:
Remind me again why we need to set it to 'forbid' instead of 'ignore'?

Reply:
I don't remember :D
I've changed this to ignore and all the tests also pass.
It might have been related to polymorphism of the TaskParameter class. 
What do you prefer?